src/components/Tables/MeetingVotesTableRow/MeetingVotesTableRow.tsx

```diff
@@ -5,11 +5,11 @@ import { TAG_CONNECTOR } from "../../../constants/StyleConstants";
 import { IndividualMeetingVote } from "../../Shared/Types/IndividualMeetingVote";
 import { VOTE_DECISION } from "../../../constants/ProjectConstants";
 import { ReactiveTableRow } from "../ReactiveTableRow";
-import { useMediaQuery } from "react-responsive";
+import useMediaQuery from "react-responsive";
 import { screenWidths } from "../../../styles/mediaBreakpoints";
 import { strings } from "../../../assets/LocalizedStrings";
 import "@mozilla-protocol/core/protocol/css/protocol.css";
-const Link = require("react-router-dom").Link;
+import { Link } from "react-router-dom";
```

Comment:
I'm surprised that the linter is not freaking out about this change.  IIRC, I had to change the Link to a `require` over an `import` because `react-router-dom` did not have its stuff properly typed.  If that has changed, that is great.

Reply:
Yep lint is all good!